source/fundamentals/crud/read-operations/sort.txt

```diff
@@ -4,135 +4,382 @@ Sort Results
+   import static com.mongodb.client.model.Sorts.ascending;
+
+   // <MongoCollection setup code here>
+   
+   collection.aggregate(Arrays.asList(Aggregates.sort(ascending("_id"))));
+
+The code snippets above both sort the documents in the
+:ref:`sample collection <sorts-crud-sort-sample>` from smallest to
+largest value of the ``_id`` field: 
+
+.. code-block:: json
+
+   {"_id": 1, "letter": "c", "food": "coffee with milk"}
+   {"_id": 2, "letter": "a", "food": "donuts and coffee"}
+   {"_id": 3, "letter": "a", "food": "maple syrup"}
+   ...
+
+In the above code snippets, we specify our sort criteria using the ``Sorts``
+builder class. While it is possible to specify sort criteria using any class
+that implements the ``Bson`` interface, we recommend that you specify sort
+criteria through the ``Sorts`` builder. For more information on the ``Sorts``
+builder class, see our
+:doc:`guide on the Sorts builder </fundamentals/builders/sort/>`.    
+
+For more information about the classes and interfaces in this section, see the
+following API documentation:
+
+- :java-docs:`FindIterable </apidocs/mongodb-driver-sync/com/mongodb/client/FindIterable.html>` 
+- :java-docs:`Aggregates </apidocs/mongodb-driver-core/com/mongodb/client/model/Aggregates.html>`
+- :java-docs:`Sorts </apidocs/mongodb-driver-core/com/mongodb/client/model/Sorts.html>`
+- :java-docs:`Bson </apidocs/bson/org/bson/conversions/Bson.html>`
+- :java-docs:`Document </apidocs/bson/org/bson/Document.html>`
+
+Sorting Direction
+-----------------
+
+The direction of your sort can either be **ascending** or **descending**.
+An ascending sort orders your results from smallest to largest. A
+descending sort orders your results from largest to smallest.
 
-    // define a cursor that can iterate through the results of the query
-    MongoCursor<Document> cursor = collection.find()
-        .sort(ascending("title")).iterator();
-    // print out items in alphabetical order
-    try {
-      while (cursor.hasNext()) {
-        System.out.println(cursor.next());
-      }
-    // close the cursor
-    } finally {
-      cursor.close();
-    }
+Here are some examples of data sorted in ascending order:
 
-This sample code prints the documents in ascending order on the ``title`` field as follows:
+* Numbers: 1, 2, 3, 43, 43, 55, 120
+* Dates: 1990-03-10, 1995-01-01, 2005-10-30, 2005-12-21 
+* Words (ASCII): Banana, Dill, carrot, cucumber, hummus
+
+Here are some examples of data sorted in descending order:
+
+* Numbers: 100, 30, 12, 12, 9, 3, 1
+* Dates: 2020-01-01, 1998-12-11, 1998-12-10, 1975-07-22 
+* Words (reverse ASCII): pear, grapes, apple, Cheese
+
+The following subsections show how to specify these sort criteria.
+
+Ascending
+~~~~~~~~~
+
+To specify an ascending sort, use the ``Sorts.ascending()`` static
+factory method. Pass the ``Sorts.ascending()`` method
+the name of the field you need to sort in ascending order.
+
+The ``ascending()`` method can be used as follows:
```

Comment:
For a moment, I thought I was looking at the builders section on Sorts and not how to use sort() because of the emphasis on using the ascending() method. I think this description needs to clearly distinguish between the functionality of sort() and ascending(). First, I'd suggest framing the code sample below as how to use the sort() method, using the ascending helper, to order the matched results.

Reply:
Thanks for the suggestion. I agree, changing this to be more explicitly about the `sort()` method is probably more helpful to learners. Fix: https://github.com/mongodb/docs-java/pull/66/commits/a53492f53a1540d0f44c053437612da8a373423d

Going forward, I will try to maintain the focus of the page and better align my writing to the expectations of readers.